### PR TITLE
[API] Adjust reset password requests

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Continuation of #12391

### CODE REVIEW

File: src/Sylius/Behat/Context/Api/Shop/LoginContext.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public function iWantToLogIn(): void
      */
     public function iWantToResetPassword(): void
     {
-        $this->request = Request::create('shop', 'request-reset-password', 'Bearer');
+        $this->request = Request::create('shop', 'reset-password-requests', 'Bearer');
```
Comment (arti0090) [inline src/Sylius/Behat/Context/Api/Shop/LoginContext.php:64]: Well I would call here also @GSadee as, first version was `reset-password-request`, then it was suggested to change to `reqest-reset-password` and PATCH `reset-password/{token}` now it is changed to `reset-password-requests` (plural form), and from what I see in our task there was suggested to use singular `reset-password-request`

Comment (GSadee) [reply src/Sylius/Behat/Context/Api/Shop/LoginContext.php:64]: To be honest, I got a little lost in what these endpoints should look like, the current form is consistent with endpoints like `apply-coupon` or `change-quantity`, but these ones should be based on the API resource, so this is a good change.

File: src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ResetPassword.xml [hunk 2]
```diff
@@ -15,15 +15,15 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
 >
-    <resource class="Sylius\Bundle\ApiBundle\Command\ResetPassword" shortName="ResetPassword">
+    <resource class="Sylius\Bundle\ApiBundle\Command\ResetPassword" shortName="ResetPasswordRequest">
```
[the remark this answers is not in the document]
Comment (GSadee) [reply src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ResetPassword.xml:18]: I would be also for changing the command name, now it is quite confusing

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Łukasz! :1st_place_medal:

**Pull request merged**